src/NSwag.CodeGeneration.CSharp/Models/CSharpClientTemplateModel.cs

```diff
@@ -129,7 +129,14 @@ public string JsonSerializerParameterCode
                 if (RequiresJsonExceptionConverter)
                     jsonConverters = jsonConverters.Concat(new[] { "JsonExceptionConverter" });
 
-                return CSharpJsonSerializerGenerator.GenerateJsonSerializerParameterCode(handleReferences, jsonConverters.ToList());
+                var parameterCode = CSharpJsonSerializerGenerator.GenerateJsonSerializerParameterCode(handleReferences, jsonConverters.ToList());
+
+                if (parameterCode.Length == 0)
+                    parameterCode = "new Newtonsoft.Json.JsonSerializerSettings()";
+                else
+                    parameterCode = parameterCode.Substring(2);
```

Comment:
1. There is a way in GenerateJsonSerializerParameterCode() when it returns an empty string, handleReference is not respected only in this case, which I think is also OK.
2. Due to probable empty string, we cannot simply return Substring(2).
3. Changing GenerateJsonSerializerParameterCode() implementation is a good idea (see below), but don't forget your current users who expect the current behavior.

I recommend the following:
* Update GenerateJsonSerializerParameterCode() by overload it:
  * Add two new parameters (just for clarification): bool initializer (when true, omit the ", " prefix)
  * Another collection for converters that cannot be instantiated but can be accessed via static properties (like the ones in NodaTime)
* The current one should call this method with false and empty collection for the new parameters

The later argument can be used in my third PR to support singleton, property accessed converters (e.g. NodaTime).

What do you think?